sound/soc/sof/control.c

```diff
@@ -303,13 +302,14 @@ int snd_sof_bytes_ext_put(struct snd_kcontrol *kcontrol,
 	/* The maximum length that can be copied is limited by IPC max
 	 * length and topology defined length for ext bytes control.
 	 */
-	if (max_length > be->max)
-		max_length = be->max;
-
-	dev_dbg(sdev->dev, "size of bytes put data is %d, max is %d\n",
-		header.length, max_length);
-	if (header.length > max_length) {
-		dev_err(sdev->dev, "error: size is too large\n");
+	dev_dbg(sdev->dev, "bytes: abi user data max size is %d\n", max_size);
+	dev_dbg(sdev->dev, "bytes: control bytes max is %d\n", be->max);
+	max_size = (be->max < max_size) ? be->max : max_size;
+	dev_dbg(sdev->dev, "bytes: size of bytes data is %d",
```

Comment:
this seems too verbose to have a debug message every time you change an EQ coefficient.
Just keep the error message with all the information needed.